web/src/inspector.js

```diff
@@ -312,7 +312,8 @@ export class Inspector extends EventTarget {
         string = `${fmt.size(verified)} (100%)`;
       }
     }
-    setTextContent(e.info.have, string);
+
+    setTextContent(e.info.have, string.includes('E2BIG') ? '...' : string);
```

Comment:
Please use the unicode ellipsis char instead:
```suggestion
    setTextContent(e.info.have, string.includes('E2BIG') ? '…' : string);
```

Reply:
This is a repeating pattern, maybe there should be a reusable sanitizer function?

```js
const sanitizeStr = (str) => ['E2BIG', 'NaN'].some((badstr) => in.includes(badstr)) ? `…` : str;
```

Reply:
@mikedld The difficulty with that is the onscreen representation, those dots can disappear into invisibility.

Reply:
Yeah, that's a better way to do it for sure — implementing and updating!